src/core/Akka.API.Tests/CoreAPISpec.cs

```diff
@@ -5,279 +5,121 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
-using System;
-using System.IO;
-using System.Linq;
-using System.Runtime.CompilerServices;
-using System.Text;
+using System.Threading.Tasks;
 using Akka.Actor;
 using Akka.Cluster;
 using Akka.Cluster.Tools.Singleton;
 using Akka.Persistence;
 using Akka.Remote;
-using Akka.Streams.Dsl;
-using ApprovalTests;
 using Xunit;
 using Akka.Persistence.Query;
 using static PublicApiGenerator.ApiGenerator;
 using Akka.Cluster.Sharding;
 using Akka.Cluster.Metrics;
 using Akka.Persistence.Query.Sql;
 using Akka.Persistence.Sql.Common.Journal;
-using ApprovalTests.Core;
-using ApprovalTests.Reporters;
-using ApprovalTests.Reporters.Mac;
-using ApprovalUtilities.Utilities;
-using DiffPlex;
-using DiffPlex.DiffBuilder;
-using DiffPlex.DiffBuilder.Model;
-using Xunit.Sdk;
-using P4MergeReporter = ApprovalTests.Reporters.P4MergeReporter;
-using P4MacMergeReporter = ApprovalTests.Reporters.Mac.P4MergeReporter;
+using Akka.Streams;
+using VerifyTests;
+using VerifyXunit;
 
 namespace Akka.API.Tests
 {
-#if(DEBUG)
-    [UseReporter(typeof(DiffPlexReporter), typeof(CustomDiffReporter), typeof(AllFailingTestsClipboardReporter))]
-#else
-    [UseReporter(typeof(DiffPlexReporter), typeof(CustomDiffReporter))]
-#endif
+    [UsesVerify]
     public class CoreAPISpec
     {
-        [Fact]
-        [MethodImpl(MethodImplOptions.NoInlining)]
-        public void ApproveCore()
+        static CoreAPISpec()
         {
-            var publicApi = Filter(GeneratePublicApi(typeof(ActorSystem).Assembly));
-            Approvals.Verify(publicApi);
+            VerifierSettings.ScrubLinesContaining("[assembly: ReleaseDateAttribute(");
```

Comment:
Very cool

Reply:
yeah the scrubbers are pretty extensive https://github.com/VerifyTests/Verify/blob/main/docs/scrubbers.md